views/partials/prs.hbs

```diff
@@ -46,5 +46,18 @@
             {{/each}}
         </div>
     {{/if}}
+
+    <div class="bg-white br2 center mt5 shadow-1 overflow-hidden w-50-ns w-80-m w-90 mt2">
+        <button class="bg-orange white center mt2 h2 bn pointer br1 pv0 ph3 db-ns" onclick='storeUser()'>This is Me</button>
+        <p class='light-silver center tc-ns'>In the future, you can find your PRs by visiting <a href='/me' class='link underline-hover'>https://hacktoberfestchecker.herokuapp.com/me</a> on this device.</p>
+    </div>
 {{/exists}}
 
+<script>
+    function storeUser() {
+        // Save username into localStorage. Recall username by visiting /me.
+        localStorage.myGitHub = document.querySelector('[name="username"').value;
+        // Provide some sort of visual feedback.
+        window.location.href='/me';
+    }
+</script>
```

Comment:
How about moving this to `main.js`?

Reply:
I was not sure if I should given the niche purpose of it.